src/parcsr_ls/par_2s_interp.c

```diff
@@ -281,27 +314,83 @@ hypre_BoomerAMGBuildModPartialExtInterpHost( hypre_ParCSRMatrix  *A,
       {
          if (CF_marker[i] == -2)
          {
-            for (j=A_diag_i[i]; j < A_diag_i[i+1]; j++)
+            /*if (num_functions > 1)
```

Comment:
This is commented out but not in the next one ext+e, which is fine. Just want to make sure it is what you want. Thanks!